src/attestation/sev.rs

```diff
@@ -8,74 +8,87 @@
 
 use serde::{Deserialize, Serialize};
 
+/// The `Finish` struct contains useful information regarding the launch
+/// configuration for the successfully launched VM.
+#[derive(Debug, Eq, PartialEq, Deserialize, Serialize)]
+pub struct Finish;
```

Comment:
Where / how is this info inside this struct?

Reply:
It isn't yet :grimacing:

Reply:
Ah, that's fair, maybe a `TODO` or something here would be appropriate until then.